docs/Releases/v0.2.0-release.md

```diff
@@ -0,0 +1,364 @@
+
+First, let's get a [Kind](https://kind.sigs.k8s.io) cluster fired up 
+
+```bash
+$ kind create cluster 
+Creating cluster "kind" ...
+ ✓ Ensuring node image (kindest/node:v1.26.3) 🖼
+ ✓ Preparing nodes 📦  
+ ✓ Writing configuration 📜 
+ ✓ Starting control-plane 🕹️ 
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-kind"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-kind
+
+Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community 🙂
+```
+
+Next, let's install the v0.2.0 release of operator-controller
+
+```bash
+$ curl -L -s https://github.com/operator-framework/operator-controller/releases/download/v0.2.0/install.sh | bash -s
+.
+.
+deployment.apps/operator-controller-controller-manager created
+deployment.apps/operator-controller-controller-manager condition met
+```
+
+If we look into the cluster, we'll see that the installation process installed the various [components](components.md) of OLM v1: 
+
+```bash
+$ kubectl get namespaces 
+NAME                         STATUS   AGE
+catalogd-system              Active   2m4s
+cert-manager                 Active   3m28s
+crdvalidator-system          Active   3m1s
+default                      Active   17m
+kube-node-lease              Active   17m
+kube-public                  Active   17m
+kube-system                  Active   17m
+local-path-storage           Active   16m
+operator-controller-system   Active   82s
+rukpak-system                Active   3m1s
+```
+
+Looks like all the [components](/docs/components.md) `catalogd-system`, `operator-controller-system`, and `rukpak-system` are available in the cluster!
+
+Let's now add a repository (catalog) of operators to the cluster, to make the operators shipped in the catalog available for installation on the cluster. We'll use the upstream community-operators catalog [quay.io/operatorhubio/catalog](https://quay.io/repository/operatorhubio/catalog?tab=tags&tag=latest): 
+
+```bash
+$ kubectl apply -f - <<EOF
+apiVersion: catalogd.operatorframework.io/v1alpha1
+kind: Catalog
+metadata:
+  name: operatorhubio
+spec:
+  source:
+    type: image
+    image:
+      ref: quay.io/operatorhubio/catalog:latest
+EOF
+```
+
+If we dig into the `catalogd-system` namespace, we can see our first difference here: 
+
+```
+$ kubectl get pods -n catalogd-system 
+NAME                                           READY   STATUS      RESTARTS   AGE
+catalogd-controller-manager-58c48974b7-rd9pw   2/2     Running     0          11m
+operatorhubio                                  0/1     Completed   0          37s
+```
+
+Looks like there are no [`registry-pods`](https://olm.operatorframework.io/docs/tasks/make-catalog-available-on-cluster/) running on cluster. 
+
+Instead, the [PackageManifest API](https://olm.operatorframework.io/docs/tasks/list-operators-available-to-install/) that was available in v0, is directly exposed, but with enhanced features: 
```

Comment:
Nit: Maybe it makes more sense to have a "note" that someone can click a "explain differences  from OLMv0" or a "Differences from OLMv0" sub section of this blog. If I was someone completely new to the Operator Framework and was getting started with OLMv1, with no knowledge of OLMv0, I may not care about the OLMv0 differences and "forcing" a read of the differences may confuse me while reading the docs.

Reply:
Removed the `enhanced features` for now. Ideally, this would mean explanation of the info available in these CRs, but I don't think it's worth spending that effort highlighting it here.